src/Microsoft.Health.Fhir.R4.Core/Features/Conformance/DefaultCapabilities.json

```diff
@@ -21499,8 +21499,16 @@
                 }
             ],
             "interaction": [
+                {
+                    "code": "batch",
+                    "documentation": "Implemented per the specification"
+                },
                 {
                     "code": "history-system"
+                },
```

Comment:
I realize this isn't your code, but are we missing `search-system` or is that expected?

Reply:
Just synched up with Brendan. This is expected.